src/Browser.php

```diff
@@ -38,6 +38,7 @@ class Browser
     const GSA = 'Google Search Appliance';
     const YANDEX = 'Yandex';
     const EDGE = 'Edge';
+    const EDGE_HTML = 'EdgeHTML';
```

Comment:
This could lead to use error, I'd prefer to still have Edge as the browser and the version be unknown if it is unknown.